region/lowernorfair/east.json

```diff
@@ -894,9 +1012,29 @@
           "nodeSubType": "blue",
           "sparking": {
             "runways": [
+              {
+                "length": 4,
+                "requires": [
+                  {"or": [
+                    "heatProof",
+                    "canHeatRun"
+                  ]},
+                  {"cost": [
+                    {"heatFrames": 85}
+                  ]}
+                ]
+              },
               {
                 "length": 12,
```

Comment:
This one will require a reset from either of the top doors, right?

Reply:
True